src/core/components/keep-alive.js

```diff
@@ -13,7 +13,7 @@ function matches (pattern: string | RegExp | Array<string>, name: string): boole
   if (Array.isArray(pattern)) {
     return pattern.indexOf(name) > -1
   } else if (typeof pattern === 'string') {
-    return pattern.split(',').indexOf(name) > -1
+    return pattern.split(',').map(i => i.trim()).indexOf(name) > -1
```

Comment:
just asking - maybe it's a good idea to trim the name too?

Reply:
makes sense, sorry for missing that, thanks.